docs/docs/Concepts/concepts-components.md

```diff
@@ -15,25 +15,32 @@ Learn more about components and how they work on this page.
 
 ## Component menu
 
-Each component is unique, but all have a menu bar at the top that looks something like the following:
+All components have a **Component menu** that contains meta settings for the component.
 
-<img src="/img/openai-model-component.png" alt="Open AI component" style={{display: 'block', margin: 'auto', width: 300}} />
+To access this menu for a specific component, select a component in a flow.
+The menu appears above the selected component.
 
-Use the component controls to do the following:
+<img src="/img/openai-model-component.png" alt="OpenAI component with the component menu visible" style={{display: 'block', margin: 'auto', width: 300}} />
 
-- **Code** — Modify the component's Python code and save your changes.
-- **Controls** — Adjust all component parameters.
-- **Freeze** — After a component runs, lock its previous output state to prevent it from re-running.
+You can use the controls in the **Component menu** to manage and configure the component on a high level, including the following actions:
 
-Click <Icon name="Ellipsis" aria-label="Horizontal ellipsis" /> **All** to see additional options for a component.
+- **Code**: Modify the component settings by directly editing the component's Python code.
+- **Controls**: Adjust all component parameters, including optional settings that are hidden by default.
+- **Tool Mode**: Enable tool mode when combining a component with an agent component.
+- **Freeze**: After a component runs, lock its previous output state to prevent it from re-running.
+- <Icon name="Ellipsis" aria-label="Horizontal ellipsis" /> **All**: See additional options for a component.
```

Comment:
Maybe being in a list is breaking it, so it can be separate from the list.

```suggestion
Click <Icon name="Ellipsis" aria-label="Horizontal ellipsis" /> **All** to see additional options for a component.
```